keyboards/bandominedoni/bandominedoni.c

```diff
@@ -0,0 +1,119 @@
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "bandominedoni.h"
+
+#ifdef RGB_MATRIX_ENABLE
+
+led_config_t g_led_config = {
+    {
+        {     75,     46,     45,     44,     43,     42, NO_LED }, \
+        {     47,     48,     49,     50,     51,     52,     53 }, \
+        {     60,     59,     58,     57,     56,     55,     54 }, \
+        {     61,     62,     63,     64,     65,     66,     67 }, \
+        {     74,     73,     72,     71,     70,     69,     68 }, \
+        { NO_LED, NO_LED, NO_LED, NO_LED, NO_LED, NO_LED, NO_LED }, \
+        {      7,      6,      4,      3,      2,      1,      5}, \
+        {      8,      9,     10,     11,     12,     13,      0 }, \
+        {     19,     18,     17,     16,     15,     14,     41 }, \
+        {     20,     21,     22,     23,     24,     25,     40 }, \
+        {     30,     29,     28,     27,     26,     39,     38 }, \
+        {     31,     32,     33,     34,     35,     36,     37 }
+    }, {
+        { 145,  6 }, { 157,  5 }, { 153, 14 }, { 149, 22 }, { 145, 32 }, { 136, 35 }, { 139, 48 }, { 134, 61 },
+        { 142, 60 }, { 148, 43 }, { 154, 29 }, { 158, 20 }, { 163, 12 }, { 168,  4 },
+        { 178,  5 }, { 173, 12 }, { 168, 20 }, { 163, 28 }, { 157, 40 }, { 151, 55 },
+        { 160, 52 }, { 166, 39 }, { 172, 28 }, { 177, 20 }, { 182, 13 }, { 188,  7 },
+        { 192, 17 }, { 186, 23 }, { 181, 30 }, { 175, 39 }, { 169, 50 },
+        { 178, 51 }, { 184, 41 }, { 190, 33 }, { 196, 27 },
+        { 199, 37 }, { 193, 45 }, { 187, 53 },
+        { 196, 56 }, { 202, 49 },
+        { 205, 61 },
+        { 202,  4 },
+        {  84, 18 }, {  73, 13 }, {  62, 10 }, {  51,  8 }, {  39, 11 }, {  24, 22 },
+        {  34, 19 }, {  45, 16 }, {  56, 16 }, {  66, 18 }, {  77, 22 }, {  87, 28 },
+        {  92, 40 }, {  81, 32 }, {  71, 27 }, {  61, 24 }, {  51, 24 }, {  41, 24 }, {  30, 28 },
+        {  24, 42 }, {  34, 37 }, {  44, 35 }, {  54, 34 }, {  64, 36 }, {  74, 38 }, {  84, 44 },
+        {  87, 54 }, {  77, 49 }, {  67, 47 }, {  57, 45 }, {  47, 45 }, {  37, 48 }, {  27, 52 }, {  19, 56 },
+    }, {
+        4, 4, 4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4,
+        4, 4, 4, 4,
+        4, 4, 4,
+        4, 4,
+        4,
+        4,
+        4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4, 4, 4
+    }
+};
+#endif
+
+#if defined(SPLIT_HAND_MATRIX_GRID)
```

Comment:
All of this is to prevent activity on the matrix scan, correct? 

If so ... you can use `MATRIX_MASK` to prevent it from actually using certain matrix positions, such ... as split hand grid  detection from registering as activity.

Reply:
The statement on the row:71 was to pass the compiling when SPLIT_HAND_MATRIX_GRID is undefined, and MASTER_RIGHT is defined, for example.
When SPLIT_HAND_MATRIX_GRID is undefined, it showed an error that 'peek_matrix_intersection()' is not used.

Reply:
Aaah, okay.